data-table/__tests__/table.js

```diff
@@ -551,3 +551,35 @@ describe('Test cell getters and setters', () => {
         expect(numErrorsVisible()).toEqual(0);
     });
 });
+
+describe("test input types", () => {
+    test("type is boolean", () => {
+        table.createDataTable({
+            'wrapperDivId': 'div-id',
+            'types': [Number, Boolean]
+        });
+        table.getCellData('div-id', 1, 1);
+        const content = document.getElementById("div-id_row_1_and_col_1_");
+        const labels = content.getElementsByClassName("dt_cell-label");
+        const label = labels[1];
+        const input = label.getElementsByTagName("input")[0];
+        expect(input.type).toBe("checkbox");
+    });
+
+});
+
+describe("test json", () => {
+    test("button", () => {
```

Comment:
Can you clarify what this is testing? It just says "test JSON" and "button" - is this testing the "print JSON" button? If so, that button is going to be removed soon - we should not be testing it